### ISSUE TITLE
Bbmap won't run with a supplied assembly and no rrnafasta

### ISSUE THREAD

**Issue by emnilsson** [LNUc-EEMiS/metatdenovo-dsl1#29]
Hi!
I'm running metatdenovo and supplying pre-trimmed and filtered reads and a pre-made assembly, which means I'm running with `skip_trimming = true` and `assembly = 'premade_assembly.fa.gz'`. Bbmap expects the contigs to be in `ch_contigs_bbmap` and reads to be in `ch_read_files_bbmap`. The contigs/assembly shouldn't be a problem, but the read channel is created in either process "trim_galore" or "skip_trimming", which at a first glance shouldn't be a problem either. But, "skip_trimming" expects to get input from `ch_read_files_remove_rrna`, which is set to empty when one does not specify `rrnafasta`. I'm doing a rerun with providing a rrnafasta-file and using skip_trimming, which should result in the proper channels to be populated, sadly this made the pipeline believe it needed to rerun all the processes so not sure it will work. I believe this to be a bug.

**Comment by erikrikarddaniel** [comment]
Yes, indeed a bug, but since the code is so messy difficult to fix. We should make a DSL2 version...
